docs/content/tutorial/custom-steps.md

```diff
@@ -0,0 +1,154 @@
+
+## Creating your app {#create-app}
+
+1. Navigate to the [app creation page](https://api.slack.com/apps/new) and select **From a manifest**.
+2. Select the workspace you want to install the application in, then click **Next**.
+3. Copy the contents of the [`manifest.json`](https://github.com/slack-samples/bolt-python-ai-chatbot/blob/main/manifest.json) file into the text box that says **Paste your manifest code here** (within the **JSON** tab) and click **Next**.
+4. Review the configuration and click **Create**.
+5. You're now in your app configuration's **Basic Information** page. Click **Install App**, then **Install to _your-workspace-name_**, then **Allow** on the screen that follows.
+
+### Obtaining and storing your environment variables {#environment-variables}
+
+Before you'll be able to successfully run the app, you'll need to obtain and set some environment variables.
+
+1. On the **Install App** page, copy your **Bot User OAuth Token**. You will store this in your environment as `SLACK_BOT_TOKEN` (we'll get to that next).
+2. Navigate to **Basic Information** and in the **App-Level Tokens** section , click **Generate Token and Scopes**. Add the [`connections:write`](https://api.slack.com/scopes/connections:write) scope, name the token, and click **Generate**. (For more details, refer to [understanding OAuth scopes for bots](https://api.slack.com/tutorials/tracks/understanding-oauth-scopes-bot)). Copy this token. You will store this in your environment as `SLACK_APP_TOKEN`.
+3. Follow [these instructions](https://confluence.atlassian.com/adminjiraserver0909/configure-an-incoming-link-1251415519.html) to create an external app link and to generate its redirect URL (the base of which will be stored as your APP_BASE_URL variable below), client ID, and client secret.
+4. Run the following commands in your terminal to store your environment variables, client ID, and client secret.
+5. You'll also need to know your team ID (found by opening your Slack instance in a web browser and copying the value within the link that starts with the letter **T**) and your app ID (found under **Basic Information**).
+
+**For macOS**
+```bash
+export SLACK_BOT_TOKEN=<your-bot-token>
+export SLACK_APP_TOKEN=<your-app-token>
+export JIRA_CLIENT_ID=<client-id>
+export JIRA_CLIENT_SECRET=<client-secret>
+```
+
+**For Windows**
+```bash
+set SLACK_BOT_TOKEN=<your-bot-token>
+set SLACK_APP_TOKEN=<your-app-token>
+set JIRA_CLIENT_ID=<client-id>
+set JIRA_CLIENT_SECRET=<client-secret>
+```
+
+## Setting up and running your local project {#configure-project}
+
+Clone the starter template onto your machine by running the following command:
+
+```bash
+git clone https://github.com/slack-samples/bolt-python-jira-functions.git
+```
+
+Change into the new project directory:
+
+```bash
+cd bolt-python-jira-functions
+```
+
+Start your Python virtual environment:
+
```

Comment:
```suggestion
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

<Tabs groupId="os">
<TabItem value="macos" label="For macOS">
```

Reply:
i'll document this type of stuff once the deno and cli are migrated -- I figured out a way to make it so we don't need to call these imports and that'll propagate through once that is merged